squad/core/models.py

```diff
@@ -1102,6 +1102,33 @@ def __init__(self, build, environment=None):
         self.has_metrics = len(values) > 0
 
 
+class BuildSummary(models.Model):
+    build = models.ForeignKey(Build, related_name='metrics_summary')
+    environment = models.ForeignKey(Environment)
+    metrics_summary = models.FloatField(null=True)
+    has_metrics = models.BooleanField(default=False)
```

Comment:
maybe we also want the test result counts here, instead of only metrics

Reply:
I've pushed a new update with tests_pass, tests_fail, tests_xfail, tests_skip. Also `BuildSummary` extends `TestSummary` now. I didn't bother keeping `regressions` and `fixes`. Should I?